public/app/features/admin/ldap/LdapSettingsPage.tsx

```diff
@@ -0,0 +1,315 @@
+        });
+      }
+      appEvents.publish({
+        type: AppEvents.alertSuccess.name,
+        payload: [t('ldap-settings-page.alert.saved', 'LDAP settings saved')],
+      });
+    } catch (error) {
+      appEvents.publish({
+        type: AppEvents.alertError.name,
+        payload: [t('ldap-settings-page.alert.error-saving', 'Error saving LDAP settings')],
+      });
+    }
+  };
+  const discardForm = async () => {
+    try {
+      setIsLoading(true);
+      await getBackendSrv().delete('/api/v1/sso-settings/ldap');
+      const payload = await getBackendSrv().get<LdapPayload>('/api/v1/sso-settings/ldap');
+      if (!payload || !payload.settings || !payload.settings.config) {
+        appEvents.publish({
+          type: AppEvents.alertError.name,
+          payload: [t('ldap-settings-page.alert.error-update', 'Error updating LDAP settings')],
+        });
+        return;
+      }
+
+      setFormSettings(payload);
+    } catch (error) {
+      appEvents.publish({
+        type: AppEvents.alertError.name,
+        payload: [t('ldap-settings-page.alert.error-saving', 'Error saving LDAP settings')],
+      });
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const documentation = (
+    <TextLink
+      href="https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/ldap/"
+      external
+    >
+      <Trans i18nKey="ldap-settings-page.documentation">documentation</Trans>
+    </TextLink>
+  );
+  const subTitle = (
+    <Trans i18nKey="ldap-settings-page.subtitle">
+      The LDAP integration in Grafana allows your Grafana users to log in with their LDAP credentials. Find out more in
+      our {documentation}.
+    </Trans>
+  );
+
+  return (
+    <Page navId="authentication" pageNav={pageNav} subTitle={subTitle}>
+      <Page.Contents>
+        <FormProvider {...methods}>
+          <form onSubmit={handleSubmit(submitAndEnableLdapSettings, onErrors)}>
+            {isLoading && <Loader />}
+            {!isLoading && formSettings && (
+              <section className={styles.form}>
+                <h3>
+                  <Trans i18nKey="ldap-settings-page.title">Basic Settings</Trans>
+                </h3>
+                <Field
+                  htmlFor="host"
+                  label={t('ldap-settings-page.host.label', 'Server host')}
+                  description={t(
+                    'ldap-settings-page.host.description',
+                    'Hostname or IP address of the LDAP server you wish to connect to.'
+                  )}
+                >
+                  <Input
+                    id="host"
+                    placeholder={t('ldap-settings-page.host.placeholder', 'example: 127.0.0.1')}
+                    type="text"
+                    {...register('settings.config.servers.0.host', { required: true })}
+                  />
+                </Field>
+                <Field
+                  htmlFor="bind-dn"
```

Comment:
You don't need to specify `htmlFor` if you have Input component with `id` as a child, it will be inferred automatically. It's mostly necessary for components used inside `Controller` or other nested children.